typings/util.d.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference types="node" />
```

Comment:
I think the container interface already described the public aspects of this, not sure if we need this in the public types